pkg/storage/mvcc.go

```diff
@@ -2171,8 +2167,11 @@ func MVCCMerge(
 	data, err := buf.marshalMeta(meta)
 	if err == nil {
 		if err = rw.Merge(metaKey, data); err == nil && ms != nil {
+			// We add only the size of value.RawBytes and do not add the
+			// mvccVersionTimestampSize as we do not incur the metadata overhead
+			// each time because each merge is into the same key.
```

Comment:
Would it make sense to add this comment here instead of at updateStatsOnMerge since this is where we are passing in `value.RawBytes`? `updateStatsOnMerge` only increments the `valSize` that is passed in.

Reply:
I think this comment could be skipped entirely, instead fleshing out the comment up above. See discussion there.